### PR TITLE
feature@spark_example

### PR DESCRIPTION
**Pull request by felipeeeantunes**
## Motivation
Adding example to show how to use `joblibspark`

## Description of the changes
This PR address this improvement https://github.com/optuna/optuna/issues/1381

### CODE REVIEW

**Comment by felipeeeantunes** [comment]
I have some doubts about it: the current version of `joblibspark` has a bug, fixed in this merge (https://github.com/joblib/joblib-spark/pull/21). I should figure out how to add it to the Dockerfile instead of using `pip`.

Also, I should figure out how to provide a `Dockerfile` and k8s' `YAML` with Spark to reproduce the example in minikube. Can you help with that?

File: examples/kubernetes/spark/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.5-slim-buster
```
Comment (crcrpar) [inline examples/kubernetes/spark/Dockerfile:1]: Could you use newer python images like 3.6 and 3.7 as we dropped the support for python 3.5, please?

File: examples/kubernetes/spark/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+WORKDIR /usr/src/
+
+RUN pip install --no-cache-dir optuna scikit-learn psycopg2-binary joblibspark
+
+
+COPY sklearn_distributed_spark.py .
+COPY check_study.sh .
```
Comment (crcrpar) [inline examples/kubernetes/spark/Dockerfile:10]: Could you add a new line to the end of this file?

**Review by crcrpar — COMMENTED** [review]
Thank you for your pull request 😃 

> I have some doubts about it: the current version of joblibspark has a bug, fixed in this merge (joblib/joblib-spark#21). I should figure out how to add it to the Dockerfile instead of using pip.

I left some comments on this part.

> Also, I should figure out how to provide a Dockerfile and k8s' YAML with Spark to reproduce the example in minikube. Can someone help with that?

Embarrassingly, I'm not quite familiar with spark.

**Comment by HideakiImamura** [comment]
Hi @felipeeeantunes! This PR seems to be still in progress. What can I do to help?

**Pull request closed without merging**